qmcpy/algorithms/function/integrand_base.py

```diff
@@ -0,0 +1,162 @@
+# I feel like having a directory named function is just asking for trouble.
+# I would consider renaming it to maybe integrands?  Or something more precise (if it makes sense).
+
+# Comments like this are not common -- the record of who did what is in GitHub
+""" Originally developed in MATLAB by Fred Hickernell. Translated to python by Sou-Cheng T. Choi and Aleksei Sorokin """
+
+# It is more standard to write np.sqrt than from numpy import sqrt
+import numpy as np
+from scipy.stats import norm
+
+# NOTE(Mike)
+# I am not a fan of direct importing, which is why I changed this function to the way it looks
+# It requires a sense of the structure of the code base that shouldn't be necessary during development
+# But maybe I just feel that way because I like defining PYTHONPATH each time I run things
+# Feel free to do this as you want
+from .. import univ_repr
+
+# Eventually, could consider congealing defaults into a constants file
+DEFAULT_NOMINAL_VALUE = 0
+
+
+class IntegrandBase(object):
```

Comment:
Could also easily just be called `Integrand` ... I don't know why I threw the `Base` in there.

Reply:
Renamed to Integrand in the most recent commit